Makefile

```diff
@@ -52,3 +52,31 @@ lint-workspace:
 
 lint:
 	@./script/lint.sh $(GOLANGCI_LINT_VERSION) "$(GOLANGCI_LINT_COMMON_OPTS)" $(GOLANGCI_LINT_DIRECTORY) "--new-from-rev=origin/main"
+
+CLPROTOS_REPO=https://github.com/smartcontractkit/chainlink-protos
+CLPROTOS_DIR=proto_vendor/chainlink-protos
+PROTOS_VERSION_FILE=.protos-version
+
+.PHONY: setup-clprotos update-clprotos clprotos
+update-clprotos: setup-clprotos
+	@cd $(CLPROTOS_DIR) && git fetch && git reset --hard $(COMMIT) && (echo $(COMMIT) > ../../$(PROTOS_VERSION_FILE))
+
+clprotos: setup-clprotos
```

Comment:
Is it a problem if .protos-version diverges from what is in go.mod?

Reply:
cc @jmank88 ?

Reply:
Nothing new here. The other protos all use pre-generated go in the chainlink-protos repository and each proto module has its own go module. 

If they also used that proto directly (they don't) it would have been an issue.